Websites/webkit.org/commit-review.md

```diff
@@ -1,14 +1,14 @@
-The WebKit project has two kinds of special status beyond being a contributor. WebKit Committers have direct read-write access to the Subversion repository, enabling them to commit changes by themselves or reviewed changes by others if authors ask committers to do so. WebKit Reviewers are permitted to review patches and may grant or deny approval for committing. Details of the review and commit process are available on the [contribution page](http://webkit.org/coding/contributing.html).
+The WebKit project has two kinds of special status beyond being a contributor. WebKit Committers have direct read-write access to the repository, enabling them to commit changes by themselves or reviewed changes by others if authors ask committers to do so. WebKit Reviewers are permitted to review patches and may grant or deny approval for committing. Details of the review and commit process are available on the [contribution page](http://webkit.org/coding/contributing.html).
```

Comment:
"have direct read-write access to the repository" is potentially misleading, as they can only commit via the merge queue. The next phrase makes it a little more specific and accurate.

Reply:
"direct" is misleading and I would support its removal if webkit-reviewers and webkit-committers also supported such removal.